core/src/main/java/com/linecorp/armeria/server/logging/LoggingServiceBuilder.java

```diff
@@ -55,4 +60,28 @@
     newDecorator() {
         return this::build;
     }
+
+    /**
+     * Sets the {@link Sampler} that determines which request needs logging.
+     */
+    public LoggingServiceBuilder sampler(Sampler<? super ServiceRequestContext> sampler) {
+        this.sampler = requireNonNull(sampler, "sampler");
+        return this;
+    }
+
+    /**
+     * Returns the {@link Sampler} that determines which request needs logging.
+     */
+    protected Sampler<? super ServiceRequestContext> sampler() {
+        return sampler;
+    }
+
+    /**
+     * Sets the rate at which to sample requests to log. Any number between {@code 0.0} and {@code 1.0} will
+     * cause a random sample of the requests to be logged.
+     */
+    public LoggingServiceBuilder samplingRate(float samplingRate) {
```

Comment:
Ditto - `sampler()` and `samplingRate()` should be moved before `build() and `newDecorator()`.